src/main/java/core/basesyntax/handler/ReturnOperationHandler.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.handler;
+
+import core.basesyntax.db.Storage;
+
+public class ReturnOperationHandler implements ShopOperationHandler {
+    @Override
+    public void doOperation(String fruitName, String quantityToOperate) {
+        Integer oldQuantityValue = Storage.FRUIT_STORAGE.get(fruitName.toLowerCase());
+        Storage.FRUIT_STORAGE.put(fruitName.toLowerCase(),
+                Integer.parseInt(quantityToOperate) + oldQuantityValue);
```

Comment:
And for return we also may assume that possibly we get returned with fruit that is not on our storage (e.g. was purchased some days ago):
```suggestion
        Integer oldQuantityValue = Storage.FRUIT_STORAGE.get(fruitName.toLowerCase());
        oldQuantityValue = oldQuantityValue == null ? 0 : oldQuantityValue;
        Storage.FRUIT_STORAGE.put(fruitName.toLowerCase(),
                Integer.parseInt(quantityToOperate) + oldQuantityValue);
```